### PR TITLE
updated online link to help docs

### PR DESCRIPTION
**Pull request by ksdani**

### CODE REVIEW

File: exchange/exchange-ps/exchange/Set-DefaultTenantBriefingConfig.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 external help file: Microsoft.Exchange.Management.RestApiClient.dll-Help.xml
 Module Name: ExchangeOnlineManagement
-online version: https://learn.microsoft.com/powershell/module/exchange/set-defaulttenantbriefingconfig
+online version: https://learn.microsoft.com/en-us/viva/insights/personal/setup/deployment-guide
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-DefaultTenantBriefingConfig.md:4]:
```suggestion
online version: https://learn.microsoft.com/viva/insights/personal/setup/deployment-guide
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ksdani, kindly commit the suggested change to remove the locale from the link. Many thanks.

File: exchange/exchange-ps/exchange/Set-DefaultTenantBriefingConfig.md (hunk 1 above)
Comment (ksdani) [reply exchange/exchange-ps/exchange/Set-DefaultTenantBriefingConfig.md:4]: done

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by chrisda** [comment]
@ksdani, sorry, that `online version:` link in the header needs to point to the cmdlet reference topic on the web. That's what makes `Get-Help <Cmdlet> -Online` work.

I added the new link you added to the RELATED LINKS section at the bottom of the topic.

**Pull request merged**